Extensions/PlatformBehavior/platformerobjectruntimebehavior.ts

```diff
@@ -1797,8 +1798,13 @@ namespace gdjs {
       this._behavior = behavior;
     }
 
-    enter() {
-      this._behavior._canJump = false;
+    enter(from: State) {
+      // canJump is used to make air jumps.
+      // It must not be reset during a jump,
+      // where "jump" means the jump + falling states.
+      if (from !== this._behavior._jumping && from !== this) {
```

Comment:
Is this possible for the Falling state (or any other state) to enter from itself?

Reply:
It's not possible, but I thought it could avoid regression if that change for some reason.
I'm fine to either remove it or add a comment.

Reply:
I've updated the comment to call this out 👍